web_ui/src/components/UsageBillingPage.tsx

```diff
@@ -513,11 +522,6 @@ function ManageSubscriptionModal({
           <Form.Group>
             <Form.Label>Billing Email </Form.Label>
             <Form.Control type="text" required disabled value={billingEmail} />
-            <Form.Text className="text-muted">
```

Comment:
We don't want to direct people to update there billing email this way because it requires updating the credit card too.